src/scss/_spacing.scss

```diff
@@ -0,0 +1,177 @@
+@if output("spacing") {
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6,
+	pre,
+	p,
+	.input--field,
+	form {
+		& + .input--field {
```

Comment:
Ik denk niet dat dit een modifyer hoeft te zijn, `.input__field` lijkt me correcterder

Reply:
Zit wat in, misschien gewoon input-field beter? Gezien het het “hoofd” component is.

Reply:
Ja lijkt me goed :)